orchagent/pfc_restore_broadcom.lua

```diff
@@ -0,0 +1,66 @@
+-- KEYS - queue IDs
```

Comment:
Is there any difference from mlnx plugin? If not, maybe it's better to rename the original to some general name and create symlinks?

Reply:
I think we will combine the detect/restore plugin into a single file in future. It might be better to separate the lua scripts for different platform for now.

Reply:
Well, now that we see that restoration criteria are common, it might be a good idea to leave them apart. @lguohan what do you think?

Reply:
I see your point. Will use pfc_restore.lua for both platforms.